content/SCALE/SCALETutorials/Apps/CommunityApps/AddStorjNode.md

```diff
@@ -15,151 +15,219 @@ keywords:
 
-Go to **Apps**, click on **Available Applications**, then scroll down to the **Storj** application and click **Install** to open the **Storj** configuration wizard.
+{{< trueimage src="/images/SCALE/Apps/LocateStorjAppWidget.png" alt="Locate Storj App Widget" id="Locate Storj App Widget" >}}
 
-{{< trueimage src="/images/SCALE/Apps/InstallApplicationsStorjSCALE.png" alt="Install Storj App SCALE" id="Install Storj App SCALE" >}}
+Click on the widget to open the **Storj** information screen.
 
-1. Accept the default name or enter a new name for your Storj application.
+{{< trueimage src="/images/SCALE/Apps/StorjAppDetailsScreen.png" alt="Storj App Information Screen" id="Storj App Information Screen" >}}
 
-   {{< hint type=note >}}
-   You can enter a name for the Storj app using lowercase alphanumeric characters that begin and end with an alphanumeric character.
-   Do not use a hyphen as the first or last character. For example, *storjnode*, or *storj-node*, but not *-storjnode* or *storjnode-*.
-   {{< /hint >}}
+Click **Install** to open the **Install Storj** app configuration wizard.
+
+{{< trueimage src="/images/SCALE/Apps/InstallStorjAppScreen.png" alt="Install Storj App SCALE" id="Install Storj App SCALE" >}}
 
-{{< trueimage src="/images/SCALE/Apps/InstallStorjAppNameSCALE.png" alt="Name Storj App SCALEL" id="Name Storj App SCALE" >}}
+Scroll down to each section or click the navigation menu item on the right of the screen to jump to that section.
 
-2. [Enter the wallet address](#getting-a-wallet-address) in **Configure Wallet for Storj**.
+Accept the default name or enter a new name for your Storj application if you prefer.
+Enter a name that consists of, and begins or ends with lowercase alphanumeric characters.
+Do not use a hyphen as the first or last character. For example, *storjnode*, or *storj-node*, but not *-storjnode* or *storjnode-*.
 
-{{< trueimage src="/images/SCALE/Apps/InstallStorjAppConfigSCALE.png" alt="Config Storj App SCALE" id="Config Storj App SCALE" >}}
+Do not change the **Version**. SCALE alerts you when a new version is available for deployed apps, and allows you to update through the click of a button on the **Installed** application screen.
 
-3. Enter the [authentication token copied from Storj](#generating-an-authentication-token-for-storj) in **Configure Auth token for Storj Node**.
-   Enter the email address associated with the token in **Configure Email for Storj**.
+1. Enter the **Storj Configuration** settings:
 
-4. Enter the [storage domain](#creating-a-ddns-host-name) (i.e., the public network DNS name) added for Storj in **Add Your Storage Domain for Storj**.
+   a. [Enter the wallet address](#getting-a-wallet-address) in **Wallet** in the **Storj Configuration** section.
+
+   {{< trueimage src="/images/SCALE/Apps/InstallStojConfigurationScreen.png" alt="Storj Configuration Settings" id="Storj Configuration Setting" >}}
```

Comment:
There's a typo in this filename also, but it's working so we can probably ignore it
```suggestion
   {{< trueimage src="/images/SCALE/Apps/InstallStojConfigurationScreen.png" alt="Storj Configuration Settings" id="Storj Configuration Settings" >}}
```